TDP/tdp-54.md

```diff
@@ -0,0 +1,98 @@
+
+Given how this is slightly different than the agreed upon process in TDP 47, an additional option will be presented in the first Snapshot done in this way to ensure everyone is happy with this approach moving forward.
+
+Similar to the funding requests, the request will stay open for 2 weeks in which Meta Governors can comment and feedback on both the process itself as well as the Cat Herder role and performance.
+
+## Tasks and responsibilities
+
+The following highlights the high level responsibilities of the Cat Herder within the wider mStableDAO ecosystem:
+
+1. Ensuring that the mStableDAOs entire governance processes are adhered by, kept up-to-date and moved forward accordingly in the ecosystem
+
+2. Creation and processing of mission critical proposals via mStable’s governance system
+
+3. Assisting & guiding contributors in crafting proposals and getting them finalized on Snapshot & on-chain
+
+4. Maintenance, curation, and tackling of open communication threads on public channels such as the forum and Discord
+
+5. Daily interaction with the community on Discord and the forum
+
+6. Organising and facilitating DAO signer calls, governance calls, and maintaining meeting minutes of them where applicable
+
+7. Making sure all proposals reach quorum and DAO signers execute transactions in a timely manner
+
+8. Managing proposal workflows to ensure they are voted on and discussed when they need to be
+
+9. Writing & posting on the official Snapshot page as a comms representative for the DAO
+
+10. Compiling and consolidating signer feedback and presenting it to Meta Governors via proposals or suggestions in Discord
+
+11. Liaising with the Product Manager of the Builder subDAO, as well as the Ops Signers of the ProtocolDAO & TreasuryDAO
+
+12. Maintaining and continuing to build out the respective internal Notion pages for the DAO
+
+13. Acting as a liaison between the wider mStable community and its Community Manager
+
+14. Business Development outreaches to other protocols & DAOs as necessary to drive the development of the mStable ecosystem and DAOs forward & help lead these opportunities being shipped into the mStable ecosystem
+
+15. Daily ecosystem research to find and investigate new opportunities & promising developments in the vertical
+
+16. Management and maintenance of the relationship between the subDAOs & the mStableDAO
+
+17. Assisting in Treasury Management for the mStableDAO
+
+18. Flexible assistance & general support function for the wider mStable ecosystem and their core contributors when required
+
+19. Quality Assurance & Quality Control that subDAOs & Signers keep delivering upon their milestones & targets, and ensuring that the public representatives of the subDAOs communicate their proof of work in time and according to the funding request
+
+## ***Compensation***

```

Comment:
H3 again? No need to make it bold if it's already a headline

Reply:
Agreed, copy pasted from bad sources 📦